client/client.go

```diff
@@ -478,11 +487,20 @@ type (
 		//  - serviceerror.NotFound
 		DescribeTaskQueue(ctx context.Context, taskqueue string, taskqueueType enumspb.TaskQueueType) (*workflowservice.DescribeTaskQueueResponse, error)
 
-		// ResetWorkflowExecution reset an existing workflow execution to WorkflowTaskFinishEventId(exclusive).
+		// ResetWorkflowExecution resets an existing workflow execution to WorkflowTaskFinishEventId(exclusive).
 		// And it will immediately terminating the current execution instance.
 		// RequestId is used to deduplicate requests. It will be autogenerated if not set.
 		ResetWorkflowExecution(ctx context.Context, request *workflowservice.ResetWorkflowExecutionRequest) (*workflowservice.ResetWorkflowExecutionResponse, error)
 
+		// UpdateWorkerBuildIDCompatability **IN DEVELOPMENT**
```

Comment:
For schedules we tagged things as `// NOTE: Experimental`. Might want to be consistent.

Reply:
Will do

Reply:
Are the struct like `UpdateWorkerBuildIDCompatabilityOptions` done or are they still in development? If they are still in development we should mark them as such

Reply:
I figured that's sort of implied by their 1:1 tie with the in-dev client functions, but in any case added the note there too :+1: